server/routers/advisor_router.py

```
@@ -129,6 +131,82 @@ def _latest_evaluation_summary(db: Session, user_id: int) -> Optional[dict[str,
+                lines.append(f"{label} : {pi[key]}")
+    ps = scv.get("professional_summary")
+    if ps and isinstance(ps, dict) and ps.get("text"):
+        lines.append(f"Résumé professionnel : {ps['text']}")
+    elif ps and isinstance(ps, str):
+        lines.append(f"Résumé professionnel : {ps}")
+    for we in scv.get("work_experience") or []:
+        if not isinstance(we, dict):
+            continue
+        role = we.get("role") or ""
+        company = we.get("company") or ""
+        dates = " – ".join(filter(None, [we.get("start_date"), we.get("end_date")]))
```

Ruff (0.14.14)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)